docs/api/extending_the_rest_api.md

```diff
@@ -155,9 +164,28 @@ The easiest way to handle cache is to re-use the `CachedValue` Value Object. It
 When you want the response to be cached, return an instance of CachedValue, with your Value Object as the argument. You can also pass a location id using the second argument, so that the Response is tagged with it:
 
 ```
-return new CachedValue($helloValue, ['locationId', 42]);
+use eZ\Publish\Core\REST\Server\Values\CachedValue;
+//...
+    public function sayHello( $name )
+    {
+        return new CachedValue(
+          new HelloValue( $name ),
+          ['locationId'=> 42]
+        );
+
+    }
+
+```
+Below you find the correspondent response header when using varnish as reverse proxy up 1.12:
```

Comment:
`Below you can find the corresponding response header when using Varnish as reverse proxy:`

I suggest removing mention of 1.12, because we version the doc through branches. We can simply replace this with info proper for 1.12 when we merge this PR to 1.12 and later branches.